injected/integration-test/test-pages/bridge/index.html

```diff
@@ -0,0 +1,58 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Document</title>
+</head>
+<body>
+<pre><code id="code"></code></pre>
+<pre><code id="code2"></code></pre>
+<button id="click">stop listening</button>
+<script type="module">
+
+    window.addEventListener = new Proxy(window.addEventListener, {
+        apply(target, thisArg, argArray) {
+            console.log('💣, addEventListener');
+            return Reflect.apply(target, thisArg, argArray)
+        }
+    })
+
+    window.dispatchEvent = new Proxy(window.dispatchEvent, {
+        apply(target, thisArg, argArray) {
+            console.log('💣, dispatchEvent');
+            return Reflect.apply(target, thisArg, argArray)
+        }
+    })
+
+    globalThis.CustomEvent = new Proxy(globalThis.CustomEvent, {
+        construct(target, argArray, newTarget) {
+            console.log('💣, CustomEvent constructor');
```

Comment:
Does this get tested at all?

Reply:
not yet, thinking today about how to test this fully - might remove these.